src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Activities/LearnerChangeOfCircumstance.cs

```diff
@@ -21,9 +21,9 @@ public LearnerChangeOfCircumstanceActivity(ICommandDispatcher commandDispatcher,
         [FunctionName(nameof(LearnerChangeOfCircumstanceActivity))]
         public async Task Update([ActivityTrigger] LearnerChangeOfCircumstanceInput input)
         {
-            _logger.LogInformation("Calling Learner Change of Circumstance for apprenticeship incentive id {apprenticeshipIncentiveId}, uln {uln}", input.ApprenticeshipIncentiveId, input.Uln);
+            _logger.LogDebug("Calling Learner Change of Circumstance for apprenticeship incentive id {apprenticeshipIncentiveId}, uln {uln}", input.ApprenticeshipIncentiveId, input.Uln);
```

Comment:
Shouldn't the log be in a decorator of the logger be a decorator of the _commandDispatcher?